src/services/wopi/utils/hostCapabilitiesHelper.js

```diff
@@ -26,8 +26,6 @@ const HOST_CAPABILITIES = [
 
 const CURRENTLY_IMPLEMENTED = [
 	'SupportsDeleteFile',
-	'SupportsLocks',
-	'SupportsGetLock',
```

Comment:
Why are this two things not more supported?

Reply:
The locks didn't work at all. They were deactivated in the basic configuration and with the new configuration it was not disabled, and threw errors when the document was opened.

Reply:
Mh okay the documentation from the creator says SupportsLocks are required.
Can you provide me with more Informations about the configuration you mention?

https://en.wikipedia.org/wiki/Web_Application_Open_Platform_Interface
https://docs.microsoft.com/en-us/microsoft-365/cloud-storage-partner-program/rest/concepts#lock
https://docs.microsoft.com/en-us/microsoft-365/cloud-storage-partner-program/rest/files/lock